crates/oxc_language_server/src/linter.rs

```diff
@@ -107,7 +110,13 @@ impl ErrorWithPosition {
                 ret_range
             },
         );
-        let source = self.miette_err.code().map_or("oxc".to_string(), |code| format!("oxc:{code}"));
+        let code = self.miette_err.code().map(|item| item.to_string());
+        let code_description = code.as_ref().and_then(|code| {
```

Comment:
If you need a doc link, maybe we add it to OxcDoagnostic directly?

Reply:
Yeah, that should be the most perfect solution. but I'm not sure if adding a new field hurt linter performance.

Reply:
CPU tends to be cheaper than memory. It all depends on this: when we provide diagnostics as JSON, do we want to make those links available? This affects use cases in external crates/applications

Reply:
Also: please consider this comment non blocking. This PR looks good to me.

Reply:
Ok, let's polish in the future.